docs/sources/microsoft-365/README.md

```diff
@@ -50,12 +50,12 @@ Neither your proxy instances nor Worklytics ever hold any API key or certificate
 
 The following Scopes are required for each connector. Note that they are all READ-only scopes.
 
-| Source&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Application Scopes                                                                                                                                                          |
-|--------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Active Directory                                                                                       | `User.Read.All` `Group.Read.All`                                                                                                                                            |
-| Calendar                                                                                               | `User.Read.All` `Group.Read.All` `OnlineMeetings.Read.All` `Calendars.Read` `MailboxSettings.Read`                                                                          |
-| Mail                                                                                                   | `User.Read.All` `Group.Read.All`  `Mail.ReadBasic.All` `MailboxSettings.Read`                                                                                               |
-| Teams                                                                                                  | `User.Read.All` `Team.ReadBasic.All` `Channel.ReadBasic.All` `Chat.Read.All` `ChannelMessage.Read.All` `CallRecords.Read.All` `OnlineMeetings.Read.All` |
+| Source&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Application Scopes                                                                                                                                                                       |
+|--------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Active Directory                                                                                       | `User.Read.All` `Group.Read.All`                                                                                                                                                         |
+| Calendar                                                                                               | `User.Read.All` `Group.Read.All` `OnlineMeetings.Read.All` `OnlineMeetingArtifact.Read.All` `Calendars.Read` `MailboxSettings.Read`                                                                                      |
```

Comment:
also for the `Calendar`? this should go in CHANGELOG, as customers using that connector who `apply` after this will see addition perms added to the app, and probably need their MSFT 365 admin to make the grant again, right??

Reply:
Yeah missing in the spec and changelog